ui/src/app/components/text-wrappers.tsx

```diff
@@ -3,3 +3,8 @@ import * as React from 'react';
 export const AouTitle = () => {
   return <span><i>All of Us</i> Research Program</span>;
 };
+
+export const AoU = () => {
```

Comment:
Can we use this above, in the AouTitle element?

Reply:
Also, weird nit: If we use `AouTitle` above, we should probably have this one be `Aou`? Just for consistency of capitalization? (As much as I don't like it because it is an acronym and we italicize it as `AoU` in text in english)

Reply:
Yeah so because we are using AouTitle for Research Program (which should be technically AoUResearchProgram) thats why i am going with AoU. 
@blrubenstein  will it be ok to use AoU here as of now and have another PR to change AouTitle to AouResearchTitle and this as AouTitle? I will be using that PR to replace  <i>All of Us</i>  used in codes to AouTitle